src/components/ICA/ICA.js

```diff
@@ -65,11 +69,29 @@ const ICA = ({
   label,
   locale,
   percentage,
+  size,
+  trending,
   truncate,
   total,
   value,
   ...other
 }) => {
+  const isSize = sizeValue => size === sizeValue;
+  const isLarge = isSize('large') || isSize('lg');
+  const isXLarge = isSize('xlarge') || isSize('xlg');
```

Comment:
For naming consistency with [Carbon's sizes](https://react.carbondesignsystem.com/?path=/docs/components-button--default#button-size):

```suggestion
  const isLarge = isSize('lg');
  const isXLarge = isSize('xl');
```